### PR TITLE
Add temporary default case to support new PointerSignalKind

### PR DESCRIPTION
**Pull request by LouiseHsu**
This PR is to unblock https://github.com/flutter/engine/pull/39637 - added a new default case to support the addition of a new PointerSignalKind member stylusAuxillaryAction. 

This PR and the engine PR addresses https://github.com/flutter/flutter/issues/73172

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

### CODE REVIEW

**Comment by Hixie** [comment]
test-exempt: code refactor with no semantic change

**Review by Hixie — APPROVED** [review]
you're missing a colon after the TODO (usual form is `// TODO(username): explanation, https://...`)

**Review by hellohuanlin — APPROVED** [review]

File: packages/flutter/lib/src/gestures/converter.dart [hunk 1]
```diff
@@ -282,6 +282,8 @@ class PointerEventConverter {
               // we don't want to return anything or miss if someone adds a new
               // enumeration to PointerSignalKind.
               throw StateError('Unreachable');
+            default: // ignore: no_default_cases, to allow adding a new [PointerSignalKind] - PointerStylusAuxiliaryAction
```
Comment (jonahwilliams) [inline packages/flutter/lib/src/gestures/converter.dart:285]: I believe you need a `return null;` here

Comment (LouiseHsu) [reply packages/flutter/lib/src/gestures/converter.dart:285]: I basically copied this verbatim from this old PR! https://github.com/flutter/flutter/pull/106906/files

Comment (hellohuanlin) [reply packages/flutter/lib/src/gestures/converter.dart:285]: that PR combines the case with the unknown case.

Comment (LouiseHsu) [reply packages/flutter/lib/src/gestures/converter.dart:285]: Interesting, okay Ill ad it just in case

**Pull request merged**